### ISSUE TITLE
Doesn't work with Android targetSdkVersion 30

### ISSUE THREAD

**Issue by nilecrocodile** [HomeXLabs/open-mail-app-flutter#10]
A message is displayed that no email application was not found.

The value of `result.didOpen` is `false` and `Result.canOpen` is `false`.

`getMailApps()` - result's length is 0.

But the gmail app is installed. And even links like `mailto:` from browser open in the gmail app.

Tested on Android 10, and Android 11.

`flutter doctor` result:
```
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 1.22.1, on Microsoft Windows [Version 10.0.19042.630], locale ru-RU)

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[!] Android Studio (not installed)
[!] VS Code (version 1.51.0)
    X Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter
[√] Connected device (1 available)
```

**Comment by MisterJimson** [comment]
Hmm maybe a permissions issue.

**Comment by MisterJimson** [comment]
Hmm I just tested on both 10 and 11 and it worked on both. Can you try the sample app in the repo and see if it works for you?

**Comment by nilecrocodile** [comment]
Yes indeed. With a newly created flutter application, it works. But not in the application created six months ago.
Maybe the matter is in the MainActivity configuration?

**Comment by nilecrocodile** [comment]
My investigation shows that if in android/app/build.gradle set versions:
```
        minSdkVersion 21
        targetSdkVersion 30

```
then the email app is not found.

And if set:

```
        minSdkVersion 16
        targetSdkVersion 29
```

then everything works.

**Comment by MisterJimson** [comment]
targetSdkVersion 30 is probably it with some new permission or something. Thanks for checking.

**Comment by nilecrocodile** [comment]
Yes, you are right. With version 30 - there is a problem. with 29 - no problem.

**Comment by martyfuhry** [comment]
It's an issue with Android 30's new package visibility privacy requirements:
https://developer.android.com/training/basics/intents/package-visibility

**Comment by martyfuhry** [comment]
I fixed it by adding an intent query to my AndroidManifest.xml:

```
<queries>
    <intent>
        <action android:name="android.intent.action.VIEW" />
        <data android:scheme="mailto" />
    </intent>
</queries>
```

**Comment by MisterJimson** [comment]
Hmm, is there a way for a library to include that so users of this package don’t need to?

**Comment by MisterJimson** [comment]
Would love a PR if so. No worries if you don’t have the time.

**Comment by MisterJimson** [comment]
Fixed in `0.0.8` and `0.1.0-nullsafety.3`.

Thank you @martyfuhry!

**Comment by martyfuhry** [comment]
No problem, happy to help.
